packages/server-renderer/src/renderToString.ts

```diff
@@ -177,7 +177,12 @@ function renderComponentSubTree(
   if (isFunction(comp)) {
     renderVNode(push, renderComponentRoot(instance), instance)
   } else {
-    if (!comp.ssrRender && !comp.render && isString(comp.template)) {
+    if (
+      !instance.render &&
```

Comment:
We should be able to just check `instance.render` here, since the component setup logic guarantees it to point to `comp.render` if it exists.

Reply:
That's right!